### ISSUE TITLE
Validation of 'limit' values too restrictive

### ISSUE THREAD

**Issue by housejester** [CenturyLinkCloud/Orchestrate-.NET-SDK#1]
Users may request a limit of 0. This is a pattern some users follow to get a count of the results, when they aren't interested in the values. We also may change the max limit, and currently do have some customers allowed to request more than the 100 default maximum. In general, clients should prefer to be lenient about parameter validation, and allow the server to reject the request. A cursory scan of this client seems to show it does that for all but the 'limit' validation :)

**Comment by jnewkirk** [comment]
I am really wondering what the value of the limit in the SDK is in this scenario. It seems to me that the only thing I should be doing is check for less than zero and specifying a number of the maximum values to return and ignore the current limit of 100. Does that make sense?

**Comment by housejester** [comment]
Yes. I would not do any validation on the 'limit' parameter (negative check is likely fine).
